smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftCodegenPlugin.kt

```diff
@@ -56,6 +57,7 @@ class SwiftCodegenPlugin : SmithyBuildPlugin {
             resolvedModel = NestedShapeTransformer.transform(resolvedModel, settings.getService(resolvedModel))
             resolvedModel = UnionIndirectivizer.transform(resolvedModel)
             resolvedModel = EquatableConformanceTransformer.transform(resolvedModel, settings.getService(resolvedModel))
+            resolvedModel = TestEquatableConformanceTransformer.transform(resolvedModel, settings.getService(resolvedModel))
```

Comment:
Performs the new model transform during codegen, along with the other model transforms.